app/src/heartbeat/schemas/logged_heartbeats.fbs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace com.google.firebase.heartbeat.cpp; // TODO - pick namespace
+
+table LoggedHeartbeats {
+    // Last date for which a heartbeat was logged
+    // YYYY-MM-DD
+    last_logged_date:string;
+
+    user_agents:[UserAgentAndDates];
```

Comment:
nit: `heartbeats:[UserAgentAndDates];` probably is more accurate.

Reply:
Done